solutions/platinum/cses-2429.mdx

```diff
@@ -0,0 +1,119 @@
+---
+id: cses-2429
+source: CSES
+title: Grid Completion
+author: Alexis Tao
+---
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^3)$
+
+<LanguageSection>
+
+<CPPSection>
+
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+typedef long long ll;
+const ll MOD = 1e9 + 7;
+
+// Compute modular inverse using Fermat's Little Theorem
+ll modInverse(ll x, ll MOD) {
```

Comment:
maybe use a javadoc style comment instead? like:

`/** @return modular inverse of x */`